### ISSUE TITLE
OnlyOffice documentation for license path

### ISSUE THREAD

**Issue by a-rode** [owncloud/docs#2492]
Could we elaborate on where to put the OnlyOffice license in https://doc.owncloud.com/server/admin_manual/appliance/configuration/office.html#how-to-install-onlyoffice ?

Short paragraph or link to https://helpcenter.onlyoffice.com/server/integration-edition/linux/linux-installation.aspx would be perfect.

**Comment by settermjd** [comment]
@a-rode, what is the priority on this?

**Comment by a-rode** [comment]
Medium-Low priority. Is important for all new customers that buy OnlyOffice through us.

Addition: This is what OnlyOffice told me.

Docker
When you are using the docker container, the license.lic file can be placed to any folder (e.g. /app/onlyoffice/DocumentServer/data/license.lic), but you will need to mount this folder when the container starts:
docker run -itd -p 80:80 --restart=always \
-v /app/onlyoffice/DocumentServer/data:/var/www/onlyoffice/Data \
onlyoffice/documentserver-ie

Linux
When you are a Debian based Linux distributive user, the license.lic file is placed to the following folder:
/var/www/onlyoffice/Data/license.lic

**Comment by settermjd** [comment]
@voroyam, can you update your documentation with this information, pls?

**Comment by voroyam** [comment]
Yes.

**Comment by settermjd** [comment]
> Yes.

Any progress on this?

**Comment by voroyam** [comment]
No. Currently busy with a univention project. As soon as that is finished I will have more time.

**Comment by mschreml** [comment]
done!

(backportet to 10.6 #3291 and 10.5 #3292)
